torch/csrc/jit/codegen/cuda/partition.cpp

```diff
@@ -52,10 +52,52 @@ static bool isFusableDevice(const Node* node) {
   return device->is_cuda();
 }
 
+inline bool isFuserSupportedType(
+    const std::shared_ptr<const TensorType>& tensor) {
+  auto scalar_type = tensor->scalarType();
+  if (scalar_type.has_value()) {
+    switch (scalar_type.value()) {
```

Comment:
Not a big fan of duplicated logic.

Can we get rid of the assert here and return `DataType::Null` instead? So we can get rid of the switch and just call `aten_to_data_type` and check if we get `DataType::Null`
https://github.com/csarofeen/pytorch/blob/20_10_20_devel/torch/csrc/jit/codegen/cuda/type.cpp#L391